algorithm/cgal/FromPolygonSoup.h

```diff
@@ -2,14 +2,18 @@
 
 #include "./repair_util.h"
 #include "./simplify_util.h"
-#include "./wrap_util.h"
 
-int FromPolygonSoup(Geometry* geometry, emscripten::val fill, bool wrap_always,
-                    double wrap_absolute_alpha, double wrap_absolute_offset,
-                    double wrap_relative_alpha, double wrap_relative_offset,
+int FromPolygonSoup(Geometry* geometry, emscripten::val fill,
                     size_t face_count_limit, double sharp_edge_threshold,
-                    bool do_remove_self_intersections, bool do_wrap,
-                    bool do_autorefine) {
+                    emscripten::val nextStrategy) {
+  std::vector<int> strategies;
+
+  for (int strategy = nextStrategy().as<int>(); strategy != -1;
```

Comment:
Ah, just lets the user tell it how to try to repair.

e.g., LoadStl(path, 'close', 'wrap')

It will try to apply each strategy in turn to repair the mesh.